builder/amazon/common/ami_config.go

```diff
@@ -18,6 +18,8 @@ type AMIConfig struct {
 	AMISkipRegionValidation bool              `mapstructure:"skip_region_validation"`
 	AMITags                 map[string]string `mapstructure:"tags"`
 	AMIEnhancedNetworking   bool              `mapstructure:"enhanced_networking"`
```

Comment:
I'm not sure we should continue to support the `enhanced_networking` option. It didn't make sense to overload it to mean both, so I don't think we should keep it around

Reply:
I'm happy with that, as long as we don't mind breaking backwards compatibility.